recipes/eigen/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+mkdir -p build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release ..
+make check
```

Comment:
Though we might have to do something here. At the rate these builds are going, I expect they are going to timeout.